packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3184,11 +3184,24 @@ export default class Meeting extends StatelessWebexPlugin {
 
   /**
    * Stops the screen share
-   * @returns {Promise} see #meetingRequest.changeMeetingFloor
+   * @returns {Promise} see #updateShare
    * @public
    * @memberof Meeting
    */
   stopShare() {
+    return this.updateShare({
+      sendShare: false,
+      receiveShare: this.mediaProperties.mediaDirection.receiveShare
+    });
+  }
+
+  /**
+   * sends stops floor request
+   * @returns {Promise} see #meetingRequest.changeMeetingFloor
+   * @private
+   * @memberof Meeting
+   */
+  _stopShare() {
```

Comment:
Could we name this more descriptive?

```suggestion
  stopFloorRequest() {
```